.github/workflows/docker.yml

```diff
@@ -5,7 +5,7 @@ on:
 
 env:
   CI_RUST_TOOLCHAIN: 1.70.0
-  REGISTRY_IMAGE: datenlord/xline
+  REPOSITORY: ${{ github.actor }}
```

Comment:
The explanation of `github.actor` in GitHub Docs is as follows.
> The username of the user that triggered the initial workflow run.  

Should we hardcode it to `datenlord`?

Reply:
In order to test in my forked repository, I haven't hard-coded the `REPOSITORY`. None of these CIs will trigger in the PRs, so should the `github.actor` here always be `xline-kv`?

Reply:
This CI may be triggered manually, So the `github.actor` should be the username of the trigger.

Reply:
Fixed